05/app.js

```diff
@@ -1,11 +1,8 @@
-import React from 'react';
-import { createRoot } from 'react-dom/client';
-
 import ContactForm from './ContactForm';
 
-const App = () => {
+function App() {
```

Comment:
Tutaj czekam na rozwiązanie ;)

Zacznij po prostu od napisania formularza, który ma parę pól i można coś do nich zapisać co jest zapisane w state. Potem jak to będzie działać to zajmij się tworzeniem walidacji.